src/Adapter/MailTemplate/MailTemplateTwigRenderer.php

```diff
@@ -156,9 +156,16 @@ private function render(
      */
     private function getMailLayoutTransformations(LayoutInterface $mailLayout, $templateType)
     {
+        $themeName = '';
```

Comment:
This is the best way I could come up to skip the application of 'CSS inline'-transformation only for `modern` template. How should this be done the Symfony way ?

Reply:
Works for me 👍